javascript/javascript_sdk/src/sensr_client.js

```diff
@@ -0,0 +1,56 @@
+const WebSocket = require('ws');
+const outputMsg = require('./../js_proto/sensr_proto/output_pb.js');
+const pointMsg = require('./../js_proto/sensr_proto/point_cloud_pb');
+
+const SensrClient = endpoint => {
+    const socket_result = new WebSocket(`ws://${endpoint}:5050`);
+    socket_result.binaryType = 'arraybuffer';
+    const socket_point = new WebSocket(`ws://${endpoint}:5051`);
+    socket_point.binaryType = 'arraybuffer';
+  
+    let objectUpdateListener, pointCloudUpdateListener;
+  
+    return {
+      listenToObjectUpdate: (streamCallback, eventCallback, timestampCallback) => {
+        if (!streamCallback && !eventCallback && !timestampCallback) return;
+        
+        if (objectUpdateListener) {
+          socket_result.removeEventListener('message', objectUpdateListener);
+          objectUpdateListener = undefined;
+        }
+  
+        objectUpdateListener = event => {
+          const response = outputMsg.OutputMessage.deserializeBinary(event.data);
+          if (response.hasStream() && streamCallback) {
```

Comment:
this checks if the response has stream and if  `streamCallback !== undefined`, right? Just want to make sure I understand this correctly